packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -235,7 +238,7 @@ export const connectEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
               connectionInfo: { key: connectionKey, url },
             }
 
-            const timeout$ = of(unsubscribe(action), subscribe(action)).pipe(
+            const timeout$ = of(subscriptionError({ ...action, message: 'WS: unsubscribe -> subscribe (unresponsive channel)' }), unsubscribe(action), subscribe(action)).pipe(
```

Comment:
Let's not confuse "message" as we call data passing through the channel a message. Either rename as `errorMsg` or `reason/type`, something different.